package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "matterfront",
+  "version": "0.1.0",
+  "description": "Mattermost frontend app powered by electron",
+  "main": "./src/main.js",
+  "devDependencies": {
+    "electron-packager": "^5.1.1",
+    "electron-prebuilt": "^0.34.2"
+  },
+  "electronVersion":"0.34.1",
+  "scripts": {
+    "start":"electron ./",
```

Comment:
This should be `src`, as I want to be able to run, build and release an unconfigured app as well as a configured one. This also makes npm start and a packaged app behave the same WRT folder hierarchy (where `src` == app.asar).